guides/common/modules/proc_deploying-a-custom-ssl-certificate-to-capsule-server.adoc

```diff
@@ -94,34 +94,12 @@ endif::[]
 +
 . Retain a copy of the `{foreman-installer}` command that the `{certs-generate}` command returns for deploying the certificate to your {SmartProxyServer}.
 +
-ifdef::satellite[]
 .Example output of `{certs-generate}`
 [options="nowrap", subs="+quotes,attributes"]
 ----
 _output omitted_
-{foreman-installer} \
---scenario capsule \
---certs-tar-file                              "_/root/capsule_certs.tar_"\
---foreman-proxy-content-parent-fqdn           "_{foreman-example-com}_"\
---foreman-proxy-register-in-foreman           "true"\
---foreman-proxy-foreman-base-url              "_https://{foreman-example-com}_"\
---foreman-proxy-trusted-hosts                 "_{foreman-example-com}_"\
---foreman-proxy-trusted-hosts                 "_{smartproxy-example-com}_"\
---foreman-proxy-oauth-consumer-key            "_s97QxvUAgFNAQZNGg4F9zLq2biDsxM7f_"\
---foreman-proxy-oauth-consumer-secret         "_6bpzAdMpRAfYaVZtaepYetomgBVQ6ehY_"\
---puppet-server-foreman-url                   "_https://{foreman-example-com}_"
-----
-endif::[]
-
-ifndef::satellite[]
-.Example output of `{certs-generate}`
-[options="nowrap", subs="+quotes,attributes"]
-----
-_output omitted_
-foreman-installer \
---scenario foreman-proxy-content \
+{installer-scenario-smartproxy} \
 --certs-tar-file                              "_/root/{smart-proxy-context}_certs.tar_"\
```

Comment:
I think we generally have a single whitespace between the last character and a backslash & a single whitespace between options and their parameters.
```suggestion
--certs-tar-file "_/root/{smart-proxy-context}_certs.tar_" \
```